packages/client/src/views/MyProfileView.vue

```diff
@@ -89,9 +94,9 @@ export default {
           description: 'Send me a daily email if new grants match my saved search(es).',
         },
         {
-          name: 'Grants Assignment',
+          name: `${shareTerminologyEnabledFlag ? 'Shared Grants' : 'Grants Assignment'}`,
           key: 'GRANT_ASSIGNMENT',
-          description: 'Send me notifications if a grant has been assigned to my USDR Grants team.',
+          description: `${shareTerminologyEnabledFlag ? grantShareDescription : grantAssignmentDescription}`,
```

Comment:
(minor nit, optional and non-blocking) The js format string with the backticks are unnecessary for the `name` and `description` here, since we're not really interpolating anything. You can just do e.g. `name: shareTerminologyEnabledFlag ? 'Shared Grants' : 'Grants Assignment', ...`